.travis.yml

```diff
@@ -0,0 +1,22 @@
+# Travis CI config file
+language: python
+python:
+    - "2.6"
+    - "2.7"
+
+addons:
+    apt:
+        packages:
+        - root-system
+        - root-plugin-hist-*
+        - libroot-*-dev
+        - libroot-bindings-python5.34
+
+before_script:
+    - .travis/install_cmsgemos.sh
+    # Copy Travis chamber info
+    - cp .travis/chamberInfo.py mapping/chamberInfo.py
```

Comment:
Looking at the file below this will work for testing.  But as a counter question how do you intend to use this to test potential changes to `chamberInfo.py` itself?

For example what happens if someone adds a dictionary to this file or changes the default behavior.  Seems like we'd then have to change `.travis/chamberInfo.py` as well to test this.

I think we should rely on using the default `mapping/chamberInfo.py` or at least one of the other links (e.g. `chamberInfo_904.py` or `chamberInfo_qc8.py` with a preference for the first).

Reply:
Your PR for `chamberInfo.py` isn't merged yet, so mine isn't built on top of it :-) The current default is for P5, meaning 12x more data in the repo, which I don't want. My plan is to move this file to `mapping/chamberInfo_Travis.py` when merging your changes.

Also, the default for 904 includes detector names, which could change at some point. I don't want the build to break just because we switched detectors, which is why I used a custom version. We can in principle mix data from different detectors in the repo (as long as they don't have different `vfatmask`s, which currently limits the number of available options).

IMHO updates to the data files should be limited to file format changes.

Reply:
I see your points.  Let's not build off unnerved PR's your right. And let's not rely on potentially transient setups.

However the detectors in P5 will not changed (even post LS2 there will be a GEMINIm27L1). And it is for the production level machines that we should be concerned with. I think ensuring the integrity tests are valid for the real system is the primary goal so let's at least have the P5 system and the /current directory for each. This would be the most representative case.

Reply:
Using P5 data would also mean that PRs that depend on file format changes would have to be tested on P5 data (or fake P5 data :-1:, or we'd have to merge with tests failing :-1:). Generating this data would slow down development a lot (it could even deadlock in case of circular dependencies).